README.md

```diff
@@ -25,9 +25,10 @@ To define a new FFT implementation in your own module, you should
 * Define a new method `AbstractFFTs.plan_fft(x, region; kws...)` that returns a `MyPlan` for at least some types of
   `x` and some set of dimensions `region`.
 
-* Define a method of `A_mul_B!(y, p::MyPlan, x)` that computes the transform `p` of `x` and stores the result in `y`.
+* Define a method of `LinearAlgebra.mul!(y, p::MyPlan, x)` that computes the transform `p` of `x` and stores the
```

Comment:
If we decide to not drop 0.6 support in this PR, should this instruct to add a method to `A_mul_B!(y, p::MyPlan, x)` or `LinearAlgebra.mul!(y, p::MyPlan, x)` depending on `VERSION`, or is there some uniform way to achieve that?

Reply:
Since Compat doesn't have `mul!` in its LinearAlgebra submodule, I don't know that it's possible to do anything other than tell people to overload `A_mul_B!` on 0.6 and `mul!` on 0.7, unfortunately.